example/lib/main.dart

```diff
@@ -17,19 +25,35 @@ class MyApp extends StatelessWidget {
             _buildComplexMarquee(),
           ].map(_wrapWithStuff).toList(),
         ),
+        floatingActionButton: FloatingActionButton.extended(
+          onPressed: () {
+            setState(() {
+              _useRtlText = !_useRtlText;
+            });
+          },
+          label: !_useRtlText ? const Text('Switch') : const Text('החלף'),
```

Comment:
I'd label them "Switch to Hebrew" and the Hebrew equivalent of "Switch to English"

Reply:
Fixed